src/schemas/chat-completion.ts

```diff
@@ -3,8 +3,10 @@ import { z } from 'zod';
 
 /** Chat completion model name */
 const ChatCompletionModel = z.union([
+  z.literal('gpt-4'),
+  z.literal('gpt-4-32k'),
   z.literal('gpt-3.5-turbo'),
-  z.literal('gpt-3.5-turbo-0301'),
+  z.literal('gpt-3.5-turbo-16k'),
   z.string(),
 ]);
```

Comment:
We can replace this with `z.string()`. The string part of the union makes the rest useless from a Zod perspective. I did it for autocompletion, but it doesn't actually work.